### PR TITLE
Register Subdomain: arnav

### PR DESCRIPTION
**Pull request by arnav-kr**
Added the domain: arnav

### CODE REVIEW

**Review by wdhdev — CHANGES_REQUESTED** [review]
404

**Review by arnav-kr — COMMENTED** [review]
Updated Details

File: domains/arnav.json [hunk 1]
```diff
@@ -6,7 +6,8 @@
       },
 
       "record": {
-        "A": ["199.36.158.100"]
+        "CNAME": ["arnav-kr.github.io"],
```
Comment (WilliamDavidHarrison) [inline domains/arnav.json:9]: CNAME cannot be an array.

File: domains/arnav.json [hunk 2]
```diff
@@ -6,7 +6,8 @@
...
+        "TXT": ["google-site-verification=BIsHFi7TKijw-rS3fXwzHQBATa3PnN0b1b09c0EsdYA"]
```
Comment (WilliamDavidHarrison) [inline domains/arnav.json:10]: Sorry, we do not support TXT verification, also TXT cannot be an array.

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: domains/arnav.json (hunk 2 above)
Comment (arnav-kr) [reply domains/arnav.json]: Oh I thought you would support it cause there was option to add a TXT record in the web client so i thought it should be work
IMG_20221007_104112

**Review by wdhdev — CHANGES_REQUESTED** [review]
Sorry, we don't support TXT verification.

**Review by arnav-kr — COMMENTED** [review]
Done

**Review by wdhdev — DISMISSED** [review]

**Review by TheBotlyNoob — DISMISSED** [review]
:+1:

**Review by wdhdev — APPROVED** [review]

**Review by MaskDuck — APPROVED** [review]

**Pull request merged**